azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -83,16 +83,7 @@ public void updateExecutableFlow(final ExecutableFlow flow)
   @Override
   public ExecutableFlow fetchExecutableFlow(final int id)
       throws ExecutorManagerException {
-    final ExecutableFlow flow = this.executionFlowDao.fetchExecutableFlow(id);
-    if (null != flow) {
-      try {
-        flow.setFlowParamsFromProps(
-            FlowLoaderUtils.loadPropsForExecutableFlow(this.projectLoader, flow));
-      } catch (ProjectManagerException e) {
-        throw new ExecutorManagerException(e);
-      }
-    }
```

Comment:
2. remove to avoid confusion, e.g.: 
when user set from UI some properties, with this code block, if the project YAML set the same properties, the UI ones (which actually take effect) got overwritten